docs/Using-Fleet/MDM-migration-guide.md

```diff
@@ -1,76 +1,69 @@
+2. [Enroll](./Adding-hosts.md) your hosts to Fleet with [Fleetd and Fleet Desktop](https://fleetdm.com/docs/using-fleet/adding-hosts#including-fleet-desktop) 
+3. Ensure your end users have access to an admin account on their Mac. End users won't be able to migrate on their own if they have a standard account.
+4. Migrate your hosts to Fleet in ABM:
+    1. In ABM, unassign the existing hosts' MDM server from the old MDM solution: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Unassign from the current MDM**, and select **Continue**.
+    2. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
+5. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
+6. Send [these guided instructions](#instructions-for-end-users) to your end users to complete the final few steps via Fleet Desktop.
+    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old MDM and when the host turns on MDM in Fleet.
 
 ## FileVault recovery keys
 
 _Available in Fleet Premium_
 
-In Fleet, you can enforce FileVault (disk encryption) to be on. If turned on, hosts’ disk encryption keys will be stored in Fleet. Learn how [here](./MDM-macOS-settings.md#disk-encryption).
+When migrating from a previous MDM, end users need to take action to escrow FileVault keys to Fleet. The **My device** page in Fleet Desktop will present users with instructions to reset their key. 
+
+To start, enforce FileVault (disk encryption) and escrow in Fleet. Learn how [here](./MDM-macOS-settings.md#disk-encryption). 
 
-During migration from your old MDM solution, disk encryption will be turned off for your macOS hosts until they are enrolled to Fleet and MDM is turned on for these hosts.
+After turning on disk encryption in Fleet, share [these guided instructions](#how-to-turn-on-disk-encryption) with your end users.
 
-If your old MDM solution enforced disk encryption, your end users will need to reset their disk encryption key for Fleet to be able to store the key. The **My device** page in Fleet Desktop will present users with instructions to reset their key. Share [these guided instructions](#how-to-turn-on-disk-encryption) with your end users.
+If your old MDM solution did not enforce disk encryption, the end user will need to restart or log out of the host.
+
+If your old MDM solution did enforce disk encryption, the end user will need to reset their disk encryption key by inputting their password, then restarting or logging out of the host. 
```

Comment:
```suggestion
If your old MDM solution did enforce disk encryption, the end user will need to reset their disk encryption key by following the prompt on the My device page and inputting their password. 
```

End user actually doesn't have to logout in this case. Up to you if we want to be more explicit about where the end user will do the pwd resetting (My device page)

Reply:
I think that's great.